### PR TITLE
Added openSUSE and SUSE Linux Enterprise support to install.sh

### PR DESCRIPTION
**Pull request by flavio**
Handle docker installation on openSUSE and SUSE Linux Enterprise via https://get.docker.io/

### CODE REVIEW

**Comment by flavio** [comment]
I think the failed test has nothing to do with my change.

**Comment by thaJeztah** [comment]
ping @tianon @jfrazelle for hack/install.sh

File: hack/install.sh [hunk 1]
```diff
@@ -133,6 +133,21 @@ do_install() {
 			exit 0
 			;;
 
+		'opensuse project'|opensuse|'suse linux'|sled)
```
Comment (tianon) [inline hack/install.sh]: heh, so many different names :smile:

**Comment by tianon** [comment]
LGTM

cc @jfrazelle

**Comment by jfrazelle** [comment]
LGTM

**Comment by flavio** [comment]
Thanks! Do you have any idea when the change will go live? I plan to use it to add openSUSE/SLE support to machine.

**Comment by jfrazelle** [comment]
it will be in the next release

**Pull request merged**